packages/cli/README.md

```diff
@@ -0,0 +1,24 @@
+# @decent/cli
+CLI for running a [Decent](https://github.com/decent-chat/decent) server alongside the standard web client, [@decent/client](https://github.com/decent-chat/decent/tree/master/packages/client). This is **not** a CLI client for Decent.
+
+If you want to access the server instance itself, check out [@decent/server](https://github.com/decent-chat/decent/tree/master/packages/server).
+
+### install
+```sh
+> npm install --global @decent/cli
+```
```

Comment:
Does this actually work? I don't want to include this in the README until it's actually published to npm.

Reply:
I'd say we publish `0.0.0` asap.